src/main/java/com/example/rets_api/controller/RetsController.java

```diff
@@ -29,8 +29,13 @@ public ResponseEntity<List<PropertyDTO>> getPropertiesByParams(@RequestBody Prop
     }
 
     @GetMapping(value="/{id}")
-    public ResponseEntity<PropertyDTO> getPropertiesById(@PathVariable("id") Long propertyId){
+    public ResponseEntity<PropertyDTO> getPropertyById(@PathVariable("id") Long propertyId){
```

Comment:
Should be properties because we'll receive a list and not just one

Reply:
correct me if I am wrong but since we are looking for a property by it's ID and the ID is unique we should only get one

Reply:
oh my, sorry! You're right!